modules/nwtc-library/src/SysIFL.f90

```diff
@@ -468,7 +468,7 @@ FUNCTION dlSym(handle,name) BIND(C,NAME="dlsym")
          DLL%ProcAddr(i) = dlSym( DLL%FileAddrX, TRIM(DLL%ProcName(i))//C_NULL_CHAR )  !the "C_NULL_CHAR" converts the Fortran string to a C-type string (i.e., adds //CHAR(0) to the end)
 
          IF(.NOT. C_ASSOCIATED(DLL%ProcAddr(i))) THEN
-            ErrStat = ErrID_Fatal
+            ErrStat = ErrID_Fatal + i - 1
```

Comment:
This is an interesting statement.  Does this mean that `ErrStat` can be larger than `ErrID_Fatal` since `NWTC_MAX_DLL_PROC = 3` in `NWTC_Base.f90`?  I guess that's useful for figuring out which ProcAddr failed.

Reply:
Yes. With the non-legacy DLL, the user is allowed to have an init routine, but it is not necessary. So, if the procedure doesn't exist, there needs to be some logic to avoid calling it and to call the normal controller routine instead. See https://github.com/OpenFAST/openfast/blob/dev/modules/servodyn/src/BladedInterface.f90#L401. I forgot to give you this code when I updated ServoDyn last time. Oops.

Reply:
Thanks for the clarification.  That makes sense.